### PR TITLE
feat: add parcel v2 oslo endpoints

### PR DESCRIPTION
**Pull request by YusufMavzer**

### CODE REVIEW

File: src/ParcelRegistry.Api.Oslo/Convertors/ParcelStatus.cs [hunk 1]
```diff
@@ -0,0 +1,23 @@
+namespace ParcelRegistry.Api.Oslo.Convertors
+{
+    using Be.Vlaanderen.Basisregisters.GrAr.Legacy.Perceel;
+
+    public static class ParcelStatusExtensions
+    {
+        public static PerceelStatus? MapToPerceelStatusSyndication(this ParcelStatus? status)
```
Comment (ArneD) [inline src/ParcelRegistry.Api.Oslo/Convertors/ParcelStatus.cs:7]: Can you check if all methods are still used?

File: src/ParcelRegistry.Api.Oslo/Infrastructure/AtomFeedConfigurationBuilder.cs [hunk 2]
```diff
@@ -0,0 +1,37 @@
+namespace ParcelRegistry.Api.Oslo.Infrastructure
```
Comment (ArneD) [inline src/ParcelRegistry.Api.Oslo/Infrastructure/AtomFeedConfigurationBuilder.cs:1]: This file doesn't seem necessary.

File: src/ParcelRegistry.Api.Oslo/Properties/launchSettings.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  },
+  "profiles": {
+    "IIS Express": {
+      "commandName": "Project",
+      "launchBrowser": true,
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Development"
+      }
+    },
+    "ParcelRegistry.Api": {
```
Comment (ArneD) [inline src/ParcelRegistry.Api.Oslo/Properties/launchSettings.json:18]: `ParcelRegistry.Api.Oslo` or `ParcelRegistry.Oslo` will be less confusing for the legacy profile.

File: src/ParcelRegistry.Api.Oslo/appsettings.json [hunk 4]
```diff
@@ -0,0 +1,43 @@
+{
+  "ConnectionStrings": {
+    "Events": "Server=(localdb)\\mssqllocaldb;Database=EFProviders.InMemory.ParcelRegistry;Trusted_Connection=True;",
```
Comment (ArneD) [inline src/ParcelRegistry.Api.Oslo/appsettings.json:3]: Not needed

File: src/ParcelRegistry.Api.Oslo/Convertors/ParcelStatus.cs (hunk 1 above)
Comment (YusufMavzer) [reply src/ParcelRegistry.Api.Oslo/Convertors/ParcelStatus.cs:7]: isn't used, removed it

File: src/ParcelRegistry.Api.Oslo/Infrastructure/AtomFeedConfigurationBuilder.cs (hunk 2 above)
Comment (YusufMavzer) [reply src/ParcelRegistry.Api.Oslo/Infrastructure/AtomFeedConfigurationBuilder.cs:1]: indeed, I removed it

**Pull request merged**